<launch>
	<include file="$(find jps_feature_matching)/launch/feature_matcher.launch" />
	<include file="$(find jps_puzzle_piece)/launch/unit_test.launch" />
	<remap from="/matched_feature_viz_node/input_image" to="/feature_matcher/homographic_transform" />
	<node pkg="jps_feature_matching" type="matched_feature_viz" name="matched_feature_viz_node" ns="matched_feature_viz_node" />
</launch>
